server/config/routes.js

```diff
@@ -54,7 +54,7 @@ module.exports = function(app, passport) {
   // fetched and seed our stores with data.
   // App is a function that requires store data and url to initialize and return the React-rendered html string
   app.get('*', function (req, res, next) {
-    App(req, res);
+    App.default(req, res);
```

Comment:
Interesting. http://stackoverflow.com/questions/33505992/babel-6-changes-how-it-exports-default. Babel 6 changes the way we export functions. Will look into this and let you know how I think we can fix this. Not a fan of `App.default`.